### PR TITLE
Restructure code with directory structure

### PR DESCRIPTION
**Pull request by jbdelmundo**
Code refactoring:

Separate files for each disclosure source
Separate files for each strategy
Separate files for each custom indicator
Separate files for each data category( PSE/Stocks, Crypto, Forex)
Separate file for backtesting wrappers backtest() for Backtrader

### CODE REVIEW

**Comment by jbdelmundo** [comment]
Will be adding changes to Readme.MD in a while

**Comment by jbdelmundo** [comment]
Old files are not yet removed

File: python/fastquant/data/crypto/crypto.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 
 def unix_time_millis(date):
-    epoch = datetime.utcfromtimestamp(0)
+    # epoch = datetime.utcfromtimestamp(0)
```
[the remark this answers is not in the document]
Comment (enzoampil) [reply python/fastquant/data/crypto/crypto.py:19]: Ah understand, it's no longer being used. Okay got it.

File: python/fastquant/data/stocks/pse.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+
+
+def datestring_to_datetime(date, sep="-"):
+    ymd = date.split(sep)
+    errmsg = "date format must be YYYY-MM-DD"
+    assert len(ymd[0]) == 4, errmsg
+    return datetime(*map(int, ymd))
+
+
+def get_stock_data(symbol, start_date, end_date, source="phisix", format="c"):
```
[the remark this answers is not in the document]
Comment (rafmacalaba) [reply python/fastquant/data/stocks/pse.py:318]: that's a good point @enzoampil, either of the two I am okay :)

**Comment by jpdeleon** [comment]
@benjamincabalona1029, why no imports in __init__.py of disclosures and and indicators, only strategies?

**Comment by benjcabalona1029** [comment]
@jpdeleon i think you meant to tag the other benjamin 😅

File: python/fastquant/data/stocks/pse.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jbdelmundo) [reply python/fastquant/data/stocks/pse.py]: Fixed! :D I added `stocks.py` for `get_stock_data()`

[the remark this answers is not in the document]
Comment (jbdelmundo) [reply python/fastquant/data/stocks/pse.py]: @enzoampil

[the remark this answers is not in the document]
Comment (enzoampil) [reply python/fastquant/data/stocks/pse.py]: awesome thanks!

**Comment by enzoampil** [comment]
Congrats on the first big PR @jbdelmundo ! :smile:

**Pull request merged**